content/manuals/dhi/core-concepts/attestations.md

```diff
@@ -38,25 +37,26 @@ They are essential for meeting industry standards such as SLSA,
 and help teams reduce the risk of supply chain attacks by making build and
 security data transparent and verifiable.
 
-## How Docker Hardened Images use attestations
+## How Docker Hardened Images and charts use attestations
```

Comment:
Images/charts do not "use" attestations :-)

I am not sure we need this section?